<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Compare_Sub_Full</class>
 <widget class="QDialog" name="Dialog_Compare_Sub_Full">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>fort.14</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="chooseFort63Sub">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="fort63Sub"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="fort14Sub"/>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sub Domain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>fort.14</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nameLineEdit_2"/>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="chooseFort14Sub">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Full Domain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>fort.63</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="chooseFort63Full">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="chooseFort14Full">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="fort63Full"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>fort.63</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="fort14Full">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>py.140</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="py140"/>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="choosePy140">
       <property name="text">
        <string>Choose...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_Compare_Sub_Full</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_Compare_Sub_Full</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
